community/pmc-tool/main/views.py

```diff
@@ -58,7 +60,22 @@ def save_data_to_database(request):
     logging.info("Verifying issues:")
     assert_equal(issue_results, issue_authors_count)
 
+    # Find authors with the first pull request in the last week
+    first_pr_authors = fetch_first_pr_authors_last_week(conn)
+
+    # Find authors with the first pull request in the last week
+    current_date = datetime.date.today()
+    one_week_ago = current_date - datetime.timedelta(days=7)
+    new_authors = []
+
+    for author, count in pr_authors_count.items():
+        first_pr_date = fetch_first_pr_date(conn, author)
+        if first_pr_date and first_pr_date.date() >= one_week_ago and count == 1:
+            new_authors.append(author)
+
     # Close the database connection
     conn.close()
 
-    return render(request, 'results.html', {'pr_results': pr_results, 'issue_results': issue_results})
+    return render(request, 'results.html', {'pr_results': pr_results, 'issue_results': issue_results,
+                                            'first_pr_authors': first_pr_authors, 'new_authors': new_authors})
```

Comment:
what is the difference between `first_pr_authors` and `new_authors`? Does it contain the same data? Seems like it does.

Reply:
first_pr_authors is all authors from the last week
new_authors are new authors who created their first PR last week